model/src/test/kotlin/licenses/LicenseInfoResolverTest.kt

```diff
@@ -281,6 +281,17 @@ class LicenseInfoResolverTest : WordSpec({
                 "(c) 2011 Holder",
                 "(c) 2012 Holder"
             )
+
```

Comment:
Nit: I guess the blank line is intended here to separate the `process = true` cases?

Reply:
Exactly.